wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -600,6 +600,8 @@ private void check() {
                     CrashReporter.saveBackgroundTrace(new RuntimeException(message), application.packageInfo());
                 }
 
+                wallet.getContext().initDashSync(getDir("masternode", MODE_PRIVATE).getAbsolutePath());
```

Comment:
This initialization used to be in onCreate and was run in a separate thread.  Now it is run inside the same of this service, before other connecting to the network.